packages/odyssey-react-mui/src/labs/AppTile.tsx

```diff
@@ -134,24 +138,50 @@ const AppTile = ({
   const tileContent = useMemo(
     () => (
       <ContentContainer>
-        <Box>
+        <Box sx={{ width: "100%" }}>
```

Comment:
`ContentContainer` is `display: flex` right? But, we only put on thing in it so, feels a bit unnecessary. If we remove `ContentContainer` then the `width: 100%` here will be unneeded.